docs/DXIL.rst

```diff
@@ -267,6 +267,17 @@ The attribute 'waveops-include-helper-lanes' is utilized to indicate that wave o
 
 'fp32-denorm-mode' is employed to define the denorm mode for the function. The possible values for this attribute can be 'any', 'preserve', or 'ftz'.
 
+Module Flags
+------------
+
+DXIL uses `Module Flags Metadata <https://releases.llvm.org/3.7.1/docs/LangRef.html#module-flags-metadata>`_ to convey additional information about the module.
```

Comment:
Rather than linking to the external LLVM doc, can we just link to the copy of the correct version of the document that lives in DXC’s tree?

Reply:
Updated.